udata_hydra_csvapi/metrics.py

```diff
@@ -22,23 +26,33 @@ async def get_object_data(session: ClientSession, model: str, sql_query: str):
     url = f"{config.PG_RST_URL}/{model}?{sql_query}"
     async with session.get(url, headers=headers) as res:
         if not res.ok:
-            handle_exception(
-                res.status, "Database error", await res.json(), None
-            )
+            handle_exception(res.status, "Database error", await res.json(), None)
         record = await res.json()
         total = process_total(res.headers.get("Content-Range"))
         return record, total
 
 
+async def get_object_data_streamed(session: ClientSession, model: str, sql_query: str):
+    headers = {"Accept": "text/csv"}
```

Comment:
This header should probably be an argument instead?

Reply:
What do you mean? The stream here makes sense only if a file is streamed no?

Reply:
Well, we may want to use `get_object_data_streamed` to stream a json file for example